CRM/Civicase/Helper/CaseCategory.php

```diff
@@ -6,7 +6,7 @@
 use CRM_Civicase_BAO_CaseCategoryInstance as CaseCategoryInstance;
 
 /**
- * CRM_Civicase_Helper_CaseCategory class.
+ * CaseCategory Helper class.
```

Comment:
shouldn't it describe the purpose of the class? and same for other similar places below